src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -63,6 +63,9 @@ internal CommandInfo(CommandBuilder builder, ModuleInfo module, CommandService s
             _action = builder.Callback;
         }
 
+        public TPrecondition GetPrecondition<TPrecondition>() where TPrecondition : PreconditionAttribute =>
```

Comment:
A command/module can't have duplicate attributes. In what case would it need to return an `IEnumerable`?